bigdata/src/main/scala/org/w3/banana/bigdata/BigdataModule.scala

```diff
@@ -0,0 +1,22 @@
+package org.w3.banana.bigdata
+
+import org.w3.banana._
+
+
+class BigdataModule  extends RDFModule with RDFOpsModule  with RecordBinderModule //with  SparqlGraphModule
+{
+
+  override type Rdf = Bigdata
```

Comment:
When I click "implement unimplemented members" Intellij Idea puts things in a
```scala
override type Rdf = ???
```
way. And actually I myself (in my libs) like putting override-s to members of abstract classes/traits that I implement, it makes the code clearer for me. However I will delete them in banana-rdf